kotlinpoet/src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -492,3 +492,12 @@ public fun Collection<CodeBlock>.joinToCode(
 public inline fun buildCodeBlock(builderAction: CodeBlock.Builder.() -> Unit): CodeBlock {
   return CodeBlock.builder().apply(builderAction).build()
 }
+
+/**
+ * Calls [CodeBlock.Builder.indent] then executes the the provided [builderAction] on the
```

Comment:
nit:
```suggestion
 * Calls [CodeBlock.Builder.indent] then executes the provided [builderAction] on the
```

Reply:
You're [a genius](https://www.reddit.com/r/explainlikeimfive/comments/3ze4ca/eli5_why_does_the_the_human_mind_ignore_the/) :joy: . Fixed!

Reply:
Officially a 10x engineer!